src/lib/server.ts

```diff
@@ -323,3 +352,14 @@ export class ZwavejsServer extends EventEmitter {
     this.logger.info(`Server closed`);
   }
 }
+
+export class EventEmitterLogTransport extends Transport {
+  public constructor(private emitter: EventEmitter) {
+    super({ format: createDefaultTransportFormat(false, false) });
+  }
+
+  public log(info: ZWaveLogInfo, next: () => void): any {
+    this.emitter.emit("logging", info);
```

Comment:
You might get away with just emitting `info.message` here. The default transport format should already have formatted it. (Please test though :) )